### ISSUE TITLE
empty template isn't there

### ISSUE THREAD

**Issue by pranaypratyush** [Odonno/surrealdb-migrations#3]
```
> RUST_BACKTRACE=full surrealdb-migrations scaffold empty
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: NotFound, message: "Path \"templates/empty\" does not exist or you don't have access!" }', /home/pranay/.cargo/registry/src/index.crates.io-6f17d22bba15001f/surrealdb-migrations-0.9.0/src/scaffold.rs:30:6
stack backtrace:
   0:     0x556a748eecfd - std::backtrace_rs::backtrace::libunwind::trace::h3e12365f7635da3e
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x556a748eecfd - std::backtrace_rs::backtrace::trace_unsynchronized::h7a6be5b9ff250f8e
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x556a748eecfd - std::sys_common::backtrace::_print_fmt::hf818e9f06250646a
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x556a748eecfd - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h5cd160069c9f3e1b
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x556a747a791f - core::fmt::write::hfa5b61a86171a98e
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/core/src/fmt/mod.rs:1254:17
   5:     0x556a748eb5d5 - std::io::Write::write_fmt::h6c6cd697a3cffe86
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/io/mod.rs:1684:15
   6:     0x556a748eead5 - std::sys_common::backtrace::_print::hf6d49b9c8ff41a80
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/sys_common/backtrace.rs:47:5
   7:     0x556a748eead5 - std::sys_common::backtrace::print::h3289230cc0e812b8
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/sys_common/backtrace.rs:34:9
   8:     0x556a748f017e - std::panicking::default_hook::{{closure}}::h15e53b47bd0b6eb4
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:271:22
   9:     0x556a748eff5e - std::panicking::default_hook::ha12d0737cacdd914
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:290:9
  10:     0x556a748f06bb - std::panicking::rust_panic_with_hook::h922001bd476c4440
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:692:13
  11:     0x556a748f05c6 - std::panicking::begin_panic_handler::{{closure}}::h7b8ff317b1305672
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:583:13
  12:     0x556a748ef206 - std::sys_common::backtrace::__rust_end_short_backtrace::h5b041c8b45cfaa82
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/sys_common/backtrace.rs:150:18
  13:     0x556a748f0362 - rust_begin_unwind
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:579:5
  14:     0x556a747a54f3 - core::panicking::panic_fmt::h1cfb4f6e7e152c39
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/core/src/panicking.rs:64:14
  15:     0x556a747a59c3 - core::result::unwrap_failed::h4ce6a6c5b54d5f95
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/core/src/result.rs:1750:5
  16:     0x556a74b1f5be - surrealdb_migrations::scaffold::main::ha5f6c259dffef8ec
  17:     0x556a74b4da1d - tokio::runtime::park::CachedParkThread::block_on::hc3670c5049e09af6
  18:     0x556a74b4e074 - tokio::runtime::scheduler::multi_thread::MultiThread::block_on::h12d496ea925d5125
  19:     0x556a74b7afd5 - tokio::runtime::runtime::Runtime::block_on::h6d43a15609401cc9
  20:     0x556a74b636e3 - surrealdb_migrations::main::hce9b347516bcd0c5
  21:     0x556a74b5c733 - std::sys_common::backtrace::__rust_begin_short_backtrace::h6267f9bbb30c5c48
  22:     0x556a74b2e5e9 - std::rt::lang_start::{{closure}}::h1e54fb892a1b0ba3
  23:     0x556a748e63b5 - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h9243ec71de803cf7
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/core/src/ops/function.rs:287:13
  24:     0x556a748e63b5 - std::panicking::try::do_call::hd0d1c25df12fa799
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:487:40
  25:     0x556a748e63b5 - std::panicking::try::h2f30a6ec993c55d8
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:451:19
  26:     0x556a748e63b5 - std::panic::catch_unwind::h486078a7c111047a
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panic.rs:140:14
  27:     0x556a748e63b5 - std::rt::lang_start_internal::{{closure}}::h6316820d1737fa0f
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/rt.rs:148:48
  28:     0x556a748e63b5 - std::panicking::try::do_call::h4c2c40d9f15bd6ed
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:487:40
  29:     0x556a748e63b5 - std::panicking::try::h565723515e5171b1
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panicking.rs:451:19
  30:     0x556a748e63b5 - std::panic::catch_unwind::hfca36f898cde3467
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/panic.rs:140:14
  31:     0x556a748e63b5 - std::rt::lang_start_internal::h91330cc7920f9b44
                               at /rustc/4a04d086cac54a41517d5657b59d5fe2caca2d71/library/std/src/rt.rs:148:20
  32:     0x556a74b637b5 - main
  33:     0x7f3a1d7fb790 - <unknown>
  34:     0x7f3a1d7fb84a - __libc_start_main
  35:     0x556a7472a9e5 - _start
  36:                0x0 - <unknown>
  ```
  
  First of all `surrealdb-migrations scaffold --template empty` doesn't work even though it's in the docs. `surrealdb-migrations scaffold empty` seems to be a valid command but there is no such template apparently. 
 The panic goes away after I do `mkdir -p templates/empty` but still everything created is empty

**Comment by Odonno** [comment]
Indeed, the right command is `surrealdb-migrations scaffold empty`. I updated the readme file accordingly.

About the bug you found, I think I have an idea. I will try to reproduce it.

**Comment by Odonno** [comment]
This should be fixed in v0.9.2. I hope you will like this project.

Sorry for the inconvenience.
